src/pages/login/index.page.tsx

```diff
@@ -1,5 +1,99 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import { SubmitErrorHandler, SubmitHandler, useForm } from 'react-hook-form';
+import { z } from 'zod';
+import Input from '@/components/Inputs/Input';
+import DatePicker from '@/components/Inputs/DatePicker';
+
+type IFormValues = {
+  email: string;
+  password: string;
+  passwordConfirm: string;
+  ReactDatepicker: Date;
+};
+
+const schema = z
+  .object({
+    email: z
+      .string()
+      .min(1, { message: '이메일을 입력해 주세요.' })
+      .email({ message: '이메일 형식으로 작성해 주세요.' }),
+    password: z
+      .string()
+      .min(1, { message: '비밀번호를 입력해 주세요.' })
```

Comment:
```suggestion
      .min(1, { message: '비밀번호를 입력해 주세요.' })
```
.min(1) 은 필수 값을 의미하는 것 같은데 저는 위클리 미션할 때 .nonempty('이메일을 입력해 주세요!') 이런 식으로 사용했는데 이건 어떠신가요?? 뭔가 .min(1), .min(8) 이렇게 되어있으니깐 어색한 느낌이 있네요

Reply:
> .min(1) 은 필수 값을 의미하는 것 같은데 저는 위클리 미션할 때 .nonempty('이메일을 입력해 주세요!') 이런 식으로 사용했는데 이건 어떠신가요?? 뭔가 .min(1), .min(8) 이렇게 되어있으니깐 어색한 느낌이 있네요

해당 부분은 nonempty 메서드가 Arrays 메서드여서 변경했습니다!
nonempty로 정하면 min(1)을 사용하라고 제안 오더라구요!
https://zod.dev/?id=nonempty